balancer-js/examples/exitExactBPTIn.ts

```diff
@@ -57,11 +57,8 @@ async function exitExactBPTIn() {
     await getBalances([pool.address, ...pool.tokensList], signer, signerAddress)
   ).map((b) => b.toString());
 
-  const { to, data, minAmountsOut } = pool.buildExitExactBPTIn(
-    signerAddress,
-    bptIn,
-    slippage
-  );
+  const { to, data, expectedAmountsOut, minAmountsOut } =
```

Comment:
Instead of `exepctedAmountsOut` should we call it `maxAmountsOut`. Expected implies its likely but that is maybe not the case (hence the slippage). Also then we have max/min.

Reply:
I'm afraid `maxAmountsOut` would be seen as a max limit, when it's not. If I'm not mistaken, it's possible to have a concurrent transaction that will result in `amountsOut` that are higher than `expectedAmountsOut`, right?
I see `expectedAmountsOut` as something that is expected or "should" happen if conditions are kept the same.
Anyway, I'm not sure if I'm just used to it, but I kinda like `expectedAmountsOut` 😅